webrtc.html

```diff
@@ -4524,7 +4524,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       given as an argument is already being sent to avoid sending
       the same <code>MediaStreamTrack</code> twice, the other ways
       do not, allowing the same <code>MediaStreamTrack</code> (possibly
-      using different <code><a>RTCRtpParameters</a></code> with different
+      using different <code><a>RTCRtpSenderParameters</a></code> with different
```

Comment:
I don't know that introducing the RTCRtpSenderParameters dictionary is really worth it. It's one more dictionary to keep track of and one more link to follow, vs. two fields that are described as "sender-only."